spec/models/cyclist_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+describe Cyclist do
+  describe "#save" do
+    context "when valid" do
+      it "creates a Cyclist user" do
+        ensure_zone_exists!
+        user = User.new(email: "cyclist@example.com")
+        cyclist = Cyclist.new(user)
+
+        saved = cyclist.save
+
+        expect(saved).to be true
+        expect(user).to be_created_with(email: "cyclist@example.com")
+        expect(cyclist).to be_created_with(email: "cyclist@example.com")
+      end
+    end
+
+    context "when invalid" do
+      it "exposes validation errors" do
+        ensure_zone_exists!
+        user = User.new(email: nil)
+        cyclist = Cyclist.new(user)
+
+        valid = cyclist.valid?
+
+        expect(valid).to be false
+        expect(cyclist.errors.keys).to eq([:email])
```

Comment:
Super nitpick: If you look for inclusion rather than equality here this test might be less brittle if you have new validation errors in the future.